x-pack/plugins/infra/public/pages/metrics/hosts_landing/enable_host_view_page.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ReactNode } from 'react';
+import { EuiPageTemplate, EuiImage, EuiSpacer } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { useEuiBackgroundColor } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useTrackPageview } from '@kbn/observability-plugin/public';
+import { MetricsPageTemplate } from '../page_template';
+import hostsLandingBeta from './hosts_landing_beta.svg';
+import { ExperimentalBadge } from '../hosts/components/experimental_badge';
+
+interface Props {
+  actions?: ReactNode;
+}
+
+export const EnableHostViewPage = ({ actions }: Props) => {
+  const backgroundColor = useEuiBackgroundColor('subdued');
+
+  useTrackPageview({ app: 'infra_metrics', path: 'hosts_feature_enable_landing_page' });
+  useTrackPageview({
+    app: 'infra_metrics',
+    path: 'hosts_feature_enable_landing_page',
+    delay: 15000,
+  });
+
+  return (
+    <MetricsPageTemplate isEmptyState>
+      <EuiPageTemplate.EmptyPrompt
+        data-test-subj="hostsLandingPage"
+        title={
+          <h2>
+            {i18n.translate('xpack.infra.hostsLandingPage.introTitle', {
```

Comment:
**nit:** I'd recommend keeping the same namespace we are using for other translations on this page to keep it consistent and well ordered: `xpack.infra.hostsViewPage`, so these translations for the landing page could be something like `xpack.infra.hostsViewPage.landing.introTitle`

Reply:
That's a good suggestion - I used another page name because I considered it a separate page with its own translation and it will be easier to remove them when we remove the page but having `.landing` is also fine. Renamed 👍